.github/actions/install-dependencies/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Install dependencies'
+description: 'Install dependencies to build sssd'
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
```

Comment:
You can indent it like:
```yaml
steps:
- shell:
```

This makes the list on the same indentation level as objects, ie:

```yaml
list:
- item1
- item2
object:
  key1:
  key2:
```

Reply:
Changed in both yml files.